docs(AwsNfsVolumeRestore): user documentation (#752)
kyma-project/cloud-manager: docs/user/resources/04-100-10-aws-nfs-volume-restore.md

# AwsNfsVolumeRestore Custom Resource

The `awsnfsvolumerestore.cloud-resources.kyma-project.io` namespaced custom resource (CR) describes the AWS EFS file 
system full restore operation on the same existing EFS file system.

> Out-of-place restore and Item-Level restore are not supported at this version of the `Cloud Manager`.

Reviewer: ```suggestion
Out-of-place restore and Item-Level restore are not supported by the `Cloud Manager`.
```
It's good practice to avoid hinting at planned features, so that we're never accidentally colliding with Product Standard [UA-030](https://wiki.one.int.sap/wiki/display/PSUA/UA-030).

While the EFS file system restore operation is running in the underlying cloud provider subscription, it needs its 

> source backup and its destination file system to be available. Upon its completion, restored file system 

Reviewer: ```suggestion
source backup and its destination file system to be available. When the restore operation is finished, the restored file system is available on a recovery directory, named `aws-backup-restore_{datetime}`, off of the root directory.
```
File paths get [code font](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#items-written-in-code-font) :)

Reviewer: What does "off of" mean? Is the recovery directory a child or sibling of the root directory?

Author: > What does "off of" mean? Is the recovery directory a child or sibling of the root directory?

AWS documents use this expression so I used it here. It means child in this case.

> would be available on a recovery directory, named aws-backup-restore_{datetime}, off of the root directory. 

Reviewer: ```suggestion
```

## Specification <!-- {docsify-ignore} -->

> This table lists the parameters of the given resource together with their descriptions:

Reviewer: This sentence adds zero value

Author: > This sentence adds zero value

This sentence appears in all other resource documents in cloud-manager repo. So I have it here for consistency. If you think it needs to be removed, I suggest a new issue is created so we remove it from all documents to be consistent. What do you think?

Reviewer: No worries, keep it for consistency. No need for an extra issue.

**Spec:**

| Parameter                        | Type    | Description                                                                                                                                                                                                        |
|----------------------------------|---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| **source**                       | object  | Required. Specifies the source backup of the restore operation.                                                                                                                                                    |

> | **source.backup**                | object  | Required. Reference of the existing AwsNfsVolumeBackup that is restored. The source volume of the AwsNfsVolumeBackup object which is referenced here, is used as the destination volume for the restore operation. |

Reviewer: ```suggestion
| **source.backup**                | object  | Required. Reference of the existing AwsNfsVolumeBackup that is restored. The source volume of the AwsNfsVolumeBackup object that is referenced here, is used as the destination volume for the restore operation. |
```
